core/src/main/java/nl/nn/adapterframework/stream/json/JsonWriter.java

```diff
@@ -138,7 +138,7 @@ public void primitive(Object value) throws SaxException {
 		try {
 			writeSeparatingComma(false);
 			if (value instanceof String) {
-				writer.write("\""+value+"\"");
+				writer.write("\""+((String)value).replaceAll("\"", "\\\\\"").replaceAll("\n", "\\\\n")+"\"");
```

Comment:
waarom gebeurd dit?

Reply:
om quotes en newlines in JSON te escapen